### ISSUE TITLE
Document that airbnb-base does not have ESLint React knowledge

### ISSUE THREAD

**Issue by eliperelman** [mozilla-neutrino/neutrino-dev#274]
Seems like removing react:recommended from the React preset when using linting has caused some rough edges because it doesn't know anything about React.

We need to either document this, or include some option to make opting in/out of react:recommended.

**Comment by metamas** [comment]
@eliperelman For the time being, how can I manually opt in to react:recommended on my project that uses `neutrino 6.0.0`. Can I add some options to my `.neutrinorc.js`? I'm unclear on what is meant by "...opting in/out of react:recommended".

**Comment by eliperelman** [comment]
@metamas right now the React preset doesn't have any ESLint knowledge. I have a patch that at least enables the react eslint plugin, but doesn't add react:recommended rules as we don't want to force that on anyone. Once this lands, you'll have 2 options for getting those rules:

1. Use a linting preset that includes it like neutrino-preset-airbnb
2. Manually add react:recommended rules such as:

```js
module.exports = {
  use: [
    ['neutrino-preset-airbnb-base', {
      eslint: {
        baseConfig: {
          extends: ['plugin:react/recommended']
        }
      }
    }],
    'neutrino-preset-react'
  ]
};
```

**Comment by eliperelman** [comment]
Should be fixed in #275. Released in v6.0.4.
